konduit-serving-api/src/main/java/ai/konduit/serving/pipeline/TransformProcessPipelineStep.java

```diff
@@ -65,15 +59,11 @@ public TransformProcessPipelineStep setOutput(Schema outputSchema) throws Except
         return (TransformProcessPipelineStep) super.setOutput("default", outputSchema);
     }
 
-    @Override
-    public TransformProcessPipelineStep setInput(String inputName, String[] columnNames, SchemaType[] types)
-            throws Exception {
+    public TransformProcessPipelineStep setInput(String inputName, String[] columnNames, SchemaType[] types) throws Exception {
```

Comment:
Always make sure to keep the at over ride annotations there. They are used for compiler level reinforcement.